vuepress/docs/.vuepress/plugins/entando-nav-version/enhanceAppFile.js

```diff
@@ -15,7 +15,7 @@ export default ({ router, isServer }) => {
         }
         //Reuse the versioned part of the path, either (/prefix)/vX.Y or (/prefix)/next
         var start = (versionPos >= 0) ? versionPos : nextPos;
-        var pos = pathname.indexOf("/", start + 1);
+        var pos = pathname.indexOf("/", start + 2);
```

Comment:
@joewhite101 , this seems off to me.

Reply:
@nshaw You might be right. this change was required in order to reflect the 6.3.2 version in the header in the dropdown of the left nav. Without it when you pick 6.3.2 from the list it says "Entando 6.3 Tutorials" Entando 6.3 Docs" even though you're on the 6.3.2 path.